### DISCUSSION TITLE
求助，请问如何修改markdown格式？

### DISCUSSION THREAD

**Discussion by YiruiLiu123** [windingwind/zotero-better-notes#754] [Q&A]
请问better notes里面支持修改markdown格式(字体，大小，颜色等)吗？用HTML来写的话太麻烦，想预先设定好格式，然后直接输入

**Comment by windingwind** [comment]
markdown没有这些格式支持，是通过html来实现的。

**Comment by windingwind** [comment]
markdown没有这些格式支持，是通过html来实现的。

**Reply by YiruiLiu123** [reply]
谢谢！我找到了将html调整的方法。
